apps/src/lib/ui/PopUpMenu.jsx

```diff
@@ -116,46 +120,84 @@ class MenuBubbleUnwrapped extends Component {
       <div style={style} className={className}>
         {this.renderMenuItems()}
         {/* These elements are used to draw the 'tail' with CSS */}
-        <span style={tailBorderStyle}/>
-        <span style={tailFillStyle}/>
+        {!this.props.minimalStyle &&
+          <span style={tailBorderStyle}/>
+        }
+        {!this.props.minimalStyle &&
+          <span style={tailFillStyle}/>
+        }
       </div>
     );
   }
 }
 export const MenuBubble = Radium(MenuBubbleUnwrapped);
 
+class MenuBreak extends Component {
+
+  render() {
+    const style = {
+      borderTop: '1px solid ' + color.lighter_gray,
+      marginTop: STANDARD_PADDING/2,
+      marginBottom: STANDARD_PADDING/2,
+      marginLeft: STANDARD_PADDING,
+      marginRight: STANDARD_PADDING,
+    };
+    return (
+      <div style={style}></div>
+    );
+  }
+}
+
+PopUpMenu.MenuBreak = Radium(MenuBreak);
+
 class Item extends Component {
   static propTypes = {
-    children: PropTypes.string.isRequired,
+    children: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.array]).isRequired,
     onClick: PropTypes.func,
+    href: PropTypes.string,
     first: PropTypes.bool,
     last: PropTypes.bool,
+    color: PropTypes.string,
+    heightDivisor: PropTypes.number,
   };
 
-  static style = {
-    color: color.dark_charcoal,
-    paddingLeft: STANDARD_PADDING,
-    paddingRight: STANDARD_PADDING,
-    cursor: 'pointer',
-    ':hover': {
-      backgroundColor: color.lightest_gray,
+  render() {
+    const {first, last, onClick, children, href, heightDivisor} = this.props;
+    if (!href && !onClick) {
+      throw new Error('Expect at least one of href/onClick');
     }
-  };
 
-  render() {
-    const {first, last, onClick, children} = this.props;
-    const style = {
-      ...PopUpMenu.Item.style,
-      paddingTop: first ? STANDARD_PADDING : STANDARD_PADDING / 2,
-      paddingBottom: last ? STANDARD_PADDING : STANDARD_PADDING / 2,
+    const paddingStyle = {
+      paddingTop: first ? STANDARD_PADDING : STANDARD_PADDING / (heightDivisor ? heightDivisor: 2),
+      paddingBottom: last ? STANDARD_PADDING : STANDARD_PADDING / (heightDivisor ? heightDivisor: 2),
+      paddingLeft: STANDARD_PADDING,
+      paddingRight: STANDARD_PADDING,
+      cursor: 'pointer',
+      ':hover': {
+        backgroundColor: color.lightest_gray,
+      }
+    };
+
+    const textStyle = {
+      color: this.props.color? this.props.color : color.dark_charcoal,
+      textDecoration: 'none', // Remove underline from anchor tags
     };
+
+    const Tag = href ? 'a' : 'div';
```

Comment:
I did not know you could do this. It feels a bit weird to have a href="null" for a div tag, but apparently this is what we do with `Button`.